packages/x-tailwindcss/.eslintrc.js

```diff
@@ -2,5 +2,8 @@ module.exports = {
   parserOptions: {
     tsconfigRootDir: __dirname,
     project: 'tsconfig.json'
+  },
+  rules: {
+    '@typescript-eslint/explicit-function-return-type': 'off'
```

Comment:
We are disabling this rule in almost every file for this package. I think makes sense just disable  it entirely. I will create a PR afterwards removing the `// estlint-disable-next-line ...` if this change goes through.